Yoast/wpseo-woocommerce
# Add productID to Schema output

## Summary

This PR can be summarized in the following changelog entry:

* Add `productID` to the schema `Product` output.

## Test instructions

This PR can be tested by following these steps:

* Go to a product page, see the product schema output contains a `productID` attribute.

Fixes #358
